README.md

```diff
@@ -1,4 +1,70 @@
-## Student life activity app - Mini Space
+<!DOCTYPE html>
+<html lang="en">
```

Comment:
This html doesn't work as we would like to.
![Screenshot from 2024-06-02 12-17-20](https://github.com/SaintAngeLs/distributed_minispace/assets/148250180/540b78d0-f484-446f-b75b-0d876173f328)

Reply:
Github html interpreter is not so good, unfortunately.